spec/integration/runner/variant-reassignment-example-8.spec.js

```diff
@@ -0,0 +1,74 @@
+ * | Product:                | Product:                                                      |                    | Product:                                                      |
+ * | slug: { en: "product" } | id: "1"                                                       |                    | id: "1"                                                       |
+ * | product-type: "pt1"     | slug: { en: "product_${timestamp1}", _ctsd: "${timestamp1}" } |                    | slug: { en: "product_${timestamp1}", _ctsd: "${timestamp1}" } |
+ * | masterVariant: v1       | product-type: "pt1"                                           |                    | product-type: "pt1"                                           |
+ * |                         | masterVariant: v1                                             |                    | masterVariant: v1                                             |
+ * |                         | variant: v2                                                   |                    |                                                               |
+ * +-------------------------+---------------------------------------------------------------+                    +---------------------------------------------------------------+
+ * |                         |                                                               |                    | Product:                                                      |
+ * |                         |                                                               |                    | id: "2"                                                       |
+ * |                         |                                                               |                    | slug: { en: "product_${timestamp2}", _ctsd: "${timestamp2}" } |
+ * |                         |                                                               |                    | product-type: "pt1"                                           |
+ * |                         |                                                               |                    | masterVariant: v2                                             |
+ * +-------------------------+---------------------------------------------------------------+--------------------+---------------------------------------------------------------+
+ */
+/* eslint-enable max-len */
+describe('Variant reassignment', () => {
+  const logger = utils.createLogger(__filename)
+  let ctpClient
+  let product1
+
+  before(async () => {
+    ctpClient = await utils.createClient()
+    const productType = await utils.ensureProductType(ctpClient)
+    const productDraft1 = utils.generateProduct(['1', '2'], productType.id)
+    product1 = await utils.ensureResource(ctpClient.products, productDraft1)
+  })
+
+  after(() =>
+    utils.deleteResourcesAll(ctpClient, logger)
+  )
+
+  it('change backup variant to valid variant', async () => {
```

Comment:
Is this test correct? It looks like you are only removing variant but not returning it back to valid ones

Reply:
What do you mean? The test is written according to our wiki. You can check the example in the class comment.

Reply:
Ahh, I see - we won't remove `_ctsd` slug, instead we let `product-import` to do it. It looks correct to me now.